recipes/lzo/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "lzo" %}
+{% set version = "2.09" %}
+{% set version_major = "2" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "f294a7ced313063c057c504257f437c8335c41bfeed23531ee4e6a2b87bcb34c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.oberhumer.com/opensource/{{ name }}/download/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    # Patch imported from https://github.com/Microsoft/vcpkg/tree/ed8e7ca79dad5b7bcf716efe25d975571e8e8c05/ports/lzo
+    - do-not-declare-setargv.patch  # [win]
```

Comment:
As far I can see in the [homepage](http://www.oberhumer.com/opensource/lzo/), there is no public issue tracker or development repository where we can report this issue.